finalPrep/modZeroHero.js

```diff
@@ -74,11 +148,50 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
+class SuperHero {
+  constructor(name, superpower, age, archNemesis, powerLevel, energyLevel) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = 'The Syntax Error';
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+
+  sayName() {
+    console.log(this.name)
+  }
+
+  maximizeEnergy() {
+    this.energyLevel = 1000
+  }
+
+  gainPower(addedPower) {
+    this.powerLevel = this.powerLevel + addedPower
+  }
+};
+
+var hero1 = new SuperHero('Light Guy', 'Gently Floating', '56');
+var hero2 = new SuperHero('Snake Girl', 'Cleaning Slow Drains', '24');
```

Comment:
You can see here, the `SuperHero` class is instantiated with only **three** arguments.